mpas_analysis/ocean/ocean_modelvsobs.py

```diff
@@ -133,6 +133,35 @@ def ocn_modelvsobs(config, field, streamMap=None, variableMap=None):
         fileOutLabel = "sstHADOI"
         unitsLabel = r'$^o$C'
 
+    elif field == 'sss':
+
+        selvals = {'nVertLevels': 0}
+
+        obs_filename = "{}/Aquarius_V3_SSS_Monthly.nc".format(obsdir)
+        dsData = xr.open_mfdataset(obs_filename)
+
+        time_start = datetime.datetime(2011, 8, 1)
+        time_end = datetime.datetime(2014, 12, 31)
+
+        ds_tslice = dsData.sel(time=slice(time_start, time_end))
+
+        # The following line converts from DASK to numpy to supress an odd
+        # warning that doesn't influence the figure output
+        ds_tslice.SSS.values
```

Comment:
@pwolfram not sure if this is helpful information, but I can also suppress this warning if I open the SSS observations with xr.open_dataset instead of xr.open_mfdataset.  It's curious that the multiple file version of open throws a warning for SSS, but not for SST or MLD.